.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: node_js
-node_js: "stable"
 
-before_install:
-  - stty cols 80
-  - npm install
```

Comment:
1. The `stty` comment wasn't doing anything valuable, as far as I'm aware
2. We were running `npm install` in the `before_install` step before running `npm ci` during the `install` step... in other words, we were duplicating dependency installation, making our builds unnecessarily long and complex! Removing this still resulted in our dependencies being installed, but nearly a minute faster 🚀 

Docs: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#npm-ci-support

Reply:
1. Not sure!
2. Nice find that's awesome!!